contracts/native/version-control/src/commands.rs

```diff
@@ -230,6 +232,26 @@ pub fn claim_namespaces(
             current: existing_namespace_count,
         });
     }
+    if namespaces_to_claim.is_empty() {
+        // Nothing to do if there is no namespace to claim
+        return Err(VCError::NoAction);
+    }
+
+    let mut fee_messages = vec![];
+    // We transfer the namespace fee if necessary
+    if !fee.amount.is_zero() {
+        let admin_account = ACCOUNT_ADDRESSES.load(deps.storage, 0)?;
+        let nb_namespaces: u128 = namespaces_to_claim.len().try_into().unwrap();
+        let necessary_fee = Coin {
```

Comment:
I would just have this be a fixed amount for now. Also if we wanted to have it be linked with the length of the namespace we should do the inverse of what's done here; shorter names are more expensive, much like how web domain names are priced.

Reply:
yeah, here the fee doesn't take into account the length of the namespaces but only the number of namespaces to claim, so exactly a fixed amount per namespace actually

Reply:
ahhh I read this wrong, sorry!